packages/datadog-plugin-aws-sdk/test/bedrock.spec.js

```diff
@@ -0,0 +1,135 @@
+      describe('with configuration', () => {
+        before(() => {
+          return agent.load('aws-sdk')
+        })
+
+        before(done => {
+          try {
+            AWS = require(`../../../versions/${bedrockRuntimeClientName}@${version}`).get()
+          } catch (e) {
+            // TODO figure out how to manage bedrock runtime client not found error
+            done()
+          }
+          bedrockRuntimeClient = new AWS.BedrockRuntimeClient(
+            { endpoint: 'http://127.0.0.1:4566', region: 'us-east-1', ServiceId: serviceName }
+          )
+          done()
+        })
+
+        after(async () => {
+          return agent.close({ ritmReset: false })
+        })
+
+        const prompt = 'What is the capital of France?'
+        const temperature = 0.5
+        const topP = 1
+        const maxTokens = 512
+        const stopSequences = []
+
+        const models = [
+          {
+            provider: 'amazon',
+            modelId: 'amazon.titan-text-lite-v1',
+            requestBody:
+              {
+                inputText: `${prompt}`,
+                textGenerationConfig: { temperature, topP, maxTokenCount: maxTokens }
+              },
+            // TODO figure out how to mock the response body instead of calling the client
+            responseBody: {}
+          },
+          // TODO add a test for the AI21 model
+          {
+            provider: 'ai21',
+            modelId: 'ai21.jamba-1-5-mini-v1:0',
+            requestBody: {},
+            responseBody: {}
+          },
+          // TODO add a test for the Anthropic model
+          {
+            provider: 'anthropic',
+            modelId: 'anthropic.claude-v2',
+            requestBody: {},
+            responseBody: {}
+          },
+          // TODO add a test for the Cohere model
+          {
+            provider: 'cohere',
+            modelId: 'cohere.command-light-text-v14',
+            requestBody: {},
+            responseBody: {}
+          },
+          // TODO add a test for the Meta model
+          {
+            provider: 'meta',
+            modelId: 'meta.llama3-70b-instruct-v1:0',
+            requestBody: {},
+            responseBody: {}
+          },
+          // TODO add a test for the Mistral model
+          {
+            provider: 'mistral',
+            modelId: 'mistral.mistral-7b-instruct-v0:2',
+            requestBody: {},
+            responseBody: {}
+          }
+        ]
+
+        models.forEach(model => {
+          it(`should invoke model for provider:${model.provider}`, done => {
+            const request = {
+              body: JSON.stringify(model.requestBody),
+              contentType: 'application/json',
+              accept: 'application/json',
+              modelId: `${model.ModelId}`
+            }
+
+            const response = {
+              body: JSON.stringify(model.responseBody)
+            }
```

Comment:
```suggestion
            const response = JSON.stringify(model.responseBody)
```
From what I can tell, we don't return a body of `{ body: response }`, it just ends up being the `response` itself. tried this locally for `amazon` models and that seemed the case, hopefully it's consistent for the other model providers 😅